packages/next/src/server/send-response.ts

```diff
@@ -27,6 +27,15 @@ export async function sendResponse(
     res.statusCode = response.status
     res.statusMessage = response.statusText
 
+    const headersWithNoMultipleValuesAllowed = [
```

Comment:
Let's inverse this list, I think it's more common that we don't want to allow multiple header values per key than it is that we do e.g. I think `Set-Cookie` is the main one and we can add more as necessary.

Reply:
done! thanks for the review